DeepCat.js

```diff
@@ -424,11 +426,31 @@
 				.replace( /"\s*$/, '' )
 				.replace( /\\(?=.)/g, '' );
 		}
+		if( /~[0-9]+$/.test ( searchTerm ) ) {
+			var split      = searchTerm.split("~");
```

Comment:
The code style is a bit off here. Please add spaces, do not use duplicate spaces, and prefer single quotes. You can also use `\d` instead of `[0-9]`.